modules/_labs/tabs/react/stories/stories.tsx

```diff
@@ -0,0 +1,112 @@
+/// <reference path="../../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import React from 'react';
+import {spacing} from '@workday/canvas-kit-react-core';
+
+import {Tabs} from '@workday/canvas-kit-labs-react-tabs';
+
+import withReadme from 'storybook-readme/with-readme';
+import README from '../README.md';
+
+export default {
+  title: 'Labs/Tabs/React',
```

Comment:
It looks like all labs stories are now under an `OTHERS` category in this PR. I think it's because the top level category got removed here. If you change this from `Labs/` to `Labs|` I think it will fix it.

![image](https://user-images.githubusercontent.com/908478/98716845-15c4d680-2341-11eb-8bf2-7411b4183e5c.png)

Reply:
Storybook 6 changes this, right?

Reply:
Yeah we'll have to change it back to `/` after SB is updated

Reply:
Fixed. For some reason, a single `Labs/*` does this for _everything_ in Labs.